guillotina/fields/annotation.py

```diff
@@ -277,25 +277,33 @@ def get_annotation_name(self, bucket_id: str) -> str:
             # get annotation for this key again as it might be the new annotation
             annotation = await self.get_annotation(context, key)
 
-        if key in annotation["keys"]:
+        insert_idx = bisect.bisect_left(annotation["keys"], key)
+        if len(annotation["keys"]) < insert_idx and annotation["keys"][insert_idx] == key:
```

Comment:
shouln't be `len(annotation["keys"]) > insert_idx`?

Reply:
You're right!